src/middlewares/sanitizeFields.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+import validator from 'validator';
+
+const fieldsToSanitize = ['firstName', 'lastName'];
+
+function removeURLs(text: string) {
+  return text
+    .split(' ')
+    .map(value => (validator.isURL(value) ? '' : value))
```

Comment:
@pettinarip I tried this out... looks like it's working well to catch URLs even if they are embedded

<img width="1716" alt="image" src="https://user-images.githubusercontent.com/54227730/217142524-c11bc346-4615-46b9-865f-bf8af84c1e73.png">

I'm noticing that these URLs are still being included in the "company"... I'm assuming somewhere we concatenate the first/last names to fill in the required company name... Just want to make sure we either sanitize this field as well, or make sure that this field isn't utilized anywhere in communications. Would suggest we just add it to the fieldsToSanitize array above.

Reply:
Nice observation. I didn't add that field because it was not used on the email but I guess it doesn't hurt us adding it in there too.